.github/workflows/trino-integration-test.yml

```diff
@@ -87,6 +87,7 @@ jobs:
         run: |
           ./gradlew -PskipTests -PtestMode=embedded -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false :trino-connector:integration-test:test
           ./gradlew -PskipTests -PtestMode=deploy -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false :trino-connector:integration-test:test
+          trino-connector/integration-test/trino-test-tools/run_test.sh
```

Comment:
What is the difference between this test script and the above?

Reply:
The script uses a different test set and testing environment to test the Trino connector. It can also specify a custom test set and testing environment for any test cases. Additionally, the above command can be modified to use the script.

Reply:
"a different test set" means tpcds and tpch, right? If so, plz add comment on it otherwise we don't know why run the script individually

Reply:
"a different test set" means the test set for Trino cascading queries. Now the test set only includes TPCH and TCCDS queries.